edx_proctoring/views.py

```diff
@@ -506,8 +515,11 @@ def get(self, request):  # pylint: disable=unused-argument
 
             if provider:
                 desktop_application_js_url = provider.get_javascript()
+                ping_interval = provider.get_ping_interval()
             else:
                 desktop_application_js_url = ''
+                ping_interval = (
```

Comment:
If there's no provider, then this is a timed exam, in which case, there's no desktop application to ping.